utils/hare-bootstrap

```diff
@@ -199,6 +199,19 @@ is_localhost() {
     [[ -e $path && $(cat $path) == $node ]] && return 0 || return 1
 }
 
+check_consul_failure() {
+
```

Comment:
Please remove, if there is extra blank line here.

Reply:
Done